### ISSUE TITLE
Given the configuration of the jupyter instance in the tutorial

### ISSUE THREAD

**Issue by LReion** [Project-MONAI/tutorials#1383]
**Is your feature request related to a problem? Please describe.**
When I ran this instance, because I didn't have enough configuration and I didn't know what kind of configuration I needed, I couldn't kill the DataLoader process

**Describe the solution you'd like**
I didn't find a solution because my config is low and I want to run through a demo locally as much as possible, but it's hard to work
**Describe alternatives you've considered**

**Comment by mingxin-zheng** [comment]
@LReion why the tutorial doesn't run?

**Comment by LReion** [comment]
spleen_segmentation_3d 
![d2ebe00478c4cf966558ce67bc7bb30](https://github.com/Project-MONAI/tutorials/assets/78899198/d44172f8-63ac-4b0a-9fba-dc8e9e64bdc8)

**Comment by mingxin-zheng** [comment]
What's the output from `print_config()`?

**Comment by mingxin-zheng** [comment]
My first guess is pytorch 2.0.1 doesn't work well with MONAI 1.1.0. We can either `pip install monai-weekly` to get the latest branch or downgrade pytorch to 1.13.

**Comment by mingxin-zheng** [comment]
The conversion to python script may skip the installation of `monai-weekly`:

![image](https://github.com/Project-MONAI/tutorials/assets/18563433/088386ba-f5c9-4c37-8fe2-7155cb40d83c)

**Comment by LReion** [comment]
I may have solved this problem, the main reason does not seem to be a compatibility problem, the main problem is that the sliding_window_inference is used to calculate when verifying, and the image and label without cropping will be loaded in this process, but because this image without cropping (2*512*512*128) and label (512*512*1) are large, it will cause the loader to use too many resources and cause the loader to crash, At the same time, the video memory will not be enough during verification
